pkg/reconciler/testing/configmap.go

```diff
@@ -94,3 +98,38 @@ func NewDataresidencyConfigMapFromRegions(regions []string) *corev1.ConfigMap {
 		},
 	}
 }
+
+// NewBrokerDeliveryConfigMapFromDeliverySpec creates a new cluster defaulted
+// broker delivery configuration map from a given delivery spec.
+func NewBrokerDeliveryConfigMapFromDeliverySpec(spec *eventingduckv1beta1.DeliverySpec) *corev1.ConfigMap {
+	configData := `
+  clusterDefaults:`
+	if spec != nil {
+		if spec.BackoffPolicy != nil {
+			configData += `
```

Comment:
My first thought was that it might be worth using a string builder here, though it seems there shouldn't be use cases when this piece is executed too often?

Reply:
I just changed it to use a string builder, I wasn't aware of the pattern when I submitted the PR :) I also adjusted the other configmap testing function `NewDataresidencyConfigMapFromRegions` to use a string builder, since that's what I was basing mine off originally.

Reply:
I see, cool!